check-vulnerabilities/ignored-safety.txt

```diff
@@ -1,4 +1,3 @@
 52495
 62044
 67599
-70612
```

Comment:
Is this intended?

Reply:
It is - this Safety advisory ahs been removed

Reply:
Nice!